src/encoded/static/components/browse/components/file-tables.js

```diff
@@ -556,28 +569,39 @@ export class RawFilesStackedTable extends React.PureComponent {
      * Much of styling/layouting is defined in CSS.
      */
     render(){
-        const { experimentSet, columnHeaders, showMetricsColumns, collapseLongLists } = this.props;
+        const { experimentSet, columnHeaders: propColHeaders, showMetricsColumns, collapseLongLists, width, preventExpand } = this.props;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['width' is missing in props validation](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4660599)